Branch hambayashi

hambayashi opened the pull request:
半林の作成したブランチのプルリクエスト

hambayashi committed:
マージ練習用にコミット

hambayashi committed:
Merge branch 'main' into branch_hambayashi

# Conflicts:
#	マージ練習/競合テスト_エクセル.xlsx
#	マージ練習/競合テスト_テキスト.txt

hambayashi committed:
バージョンの訂正

fujikomatsu commented on マージ練習/競合テスト_エクセル.xlsx:
■指摘
・橋山さんの苗字と名前の間に無駄な空白がたくさんあるので削除をしてください
・石井ゆさんの名字が間違っているので修正してください
・半林さんの名前も入れてください。

fujikomatsu replied:
修正を確認しました。

fujikomatsu commented on マージ練習/競合テスト_テキスト.txt:
こちらは問題なしです。

fujikomatsu reviewed (commented):
レビュー完了しました。指摘を反映してください。

hambayashi committed:
レビューコメントに対応
エクセルファイルに以下の変更を追加
・橋山さんの苗字と名前の間に無駄な空白がたくさんあるので削除
・石井ゆさんの名字が間違っているので修正
・半林の名前も追加

hambayashi commented:
修正完了しました。確認お願いします。
